src/migration-scripts/firewall/6-to-7

```diff
@@ -107,13 +107,26 @@ icmpv6_translations = {
     'unknown-option': [4, 2]
 }
 
+v4_found = False
+v6_found = False
+v4_groups = ["address-group", "network-group", "port-group"]
+v6_groups = ["ipv6-address-group", "ipv6-network-group", "port-group"]
+
 if config.exists(base + ['group']):
     for group_type in config.list_nodes(base + ['group']):
         for group_name in config.list_nodes(base + ['group', group_type]):
             name_description = base + ['group', group_type, group_name, 'description']
             if config.exists(name_description):
                 tmp = config.return_value(name_description)
                 config.set(name_description, value=tmp[:max_len_description])
+            if '+' in group_name:
+                if group_type in v4_groups and not v4_found:
+                    v4_found = True
+                if group_type in v6_groups and not v6_found:
+                    v6_found = True
+                new_group_name = group_name.replace('+', '_')
```

Comment:
What if the config has both `LAN+VLANS` **and** `LAN_VLANS` groups already? :)

It's a rather unlikely situation, but I think we should agree what exactly we should do if our normalized name already exists in the old config.

Reply:
Iv'e submitted a patch for this situation